Dockerfile.manager

```
@@ -0,0 +1,13 @@
+FROM comfyui-sos-comfyui:latest
```

Hadolint (2.15.1)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)